https://issues.redhat.com/browse/ACM-25350--upgrade changes and impro… (#8373)
open-cluster-management/rhacm-docs: install/install_connected.adoc

> 

Author: This change is added to install.

* Your {ocp} cluster must have access to the {acm-short} operator in the OperatorHub catalog from the {ocp-short} console. 

> * Ensure that your {ocp-short} cluster is supported by your current {acm-short} version. See the link:https://access.redhat.com/articles/7133095[{acm-short} Support Matrix] for more information about the required {ocp-short} versions for each {acm-short} version.

Reviewer: I like your earlier suggestion to use compatible. What do you think about also using it here? 
```suggestion
* Ensure that your {ocp-short} cluster is compatible with your current {acm-short} version. See the link:https://access.redhat.com/articles/7133095[{acm-short} Support Matrix] for more information about the required {ocp-short} versions for each {acm-short} version.
```

Author: Decided to keep as is bc that term is what we are using in other areas of the doc to show that the customer CAN upgrade but if they do, they don't get support for any errors.
